sklearn/decomposition/pca.py

```diff
@@ -279,31 +279,31 @@ class PCA(_BasePCA):
 
     Examples
     --------
-    >>> import numpy as np
-    >>> from sklearn.decomposition import PCA
-    >>> X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
-    >>> pca = PCA(n_components=2)
-    >>> pca.fit(X)
+    import numpy as np
+    from sklearn.decomposition import PCA
+    X = np.array([[-1, -1], [-2, -1], [-3, -2], [1, 1], [2, 1], [3, 2]])
+    pca = PCA(n_components=2)
+    pca.fit(X)
```

Comment:
Please revert these changes. `>>>` indicates that it's a doctring and will be run as part of the test suite.

Reply:
My bad! Sorry about that